docs/resources/project_badge.md

```diff
@@ -24,6 +24,7 @@ resource "gitlab_project_badge" "example" {
   project   = gitlab_project.foo.id
   link_url  = "https://example.com/badge-123"
   image_url = "https://example.com/badge-123.svg"
+  name      = "badge-123"
```

Comment:
This is actually a generated file and the content origins from https://github.com/gitlabhq/terraform-provider-gitlab/blob/main/examples/resources/gitlab_project_badge/resource.tf

Please change it there and run `make reviewable` to generate the docs :)

Reply:
Fixed